### ISSUE TITLE
`test_subtitles.py` fails due to changes in FATE tests from ffmpeg

### ISSUE THREAD

**Issue by yu17** [PyAV-Org/PyAV#1229]
## Overview

The expected output of the FATE test suite has been changed, due to which `test_subtitles.py` now fails.

Related commits from ffmpeg:
[avcodec/pnm: explicitly tag color range](https://git.ffmpeg.org/gitweb/ffmpeg.git/commit/96d2a40b9ee0ddd1844b82c23a074804c316caa5)
[fftools/ffmpeg: rework keeping track of file duration for -stream_loop](https://git.ffmpeg.org/gitweb/ffmpeg.git/commit/889a022cceea88072e3b9716af3d7c795728f5c5)

## Expected behavior

Running `python -m pytest` at the end of the build should produce no errors.

## Actual behavior

```
__________________________________ TestSubtitle.test_movtext ___________________________________
...

        sub = subset[0]
        self.assertIsInstance(sub, AssSubtitle)
>       self.assertEqual(sub.ass, "0,0,Default,,0,0,0,,- Test 1.\\N- Test 2.")
E       AssertionError: 'Dialogue: 0,0:00:00.97,0:00:02.54,Default,,0,0,0,,- Test 1.\\N- Test 2.\r\n' != '0,0,Default,,0,0,0,,- Test 1.\\N- Test 2.'
E       - Dialogue: 0,0:00:00.97,0:00:02.54,Default,,0,0,0,,- Test 1.\N- Test 2.
E       + 0,0,Default,,0,0,0,,- Test 1.\N- Test 2.

tests/test_subtitles.py:26: AssertionError
=================================== short test summary info ====================================
FAILED tests/test_subtitles.py::TestSubtitle::test_movtext - AssertionError: 'Dialogue: 0,0:00:00.97,0:00:02.54,Default,,0,0,0,,- Test 1.\\N- Test 2.\r\...
```

## Investigation

As stated above, the failure seems to be caused by changes in ffmpeg. Replacing the assertion with the correct result should fix the issue.

**Comment by WyattBlue** [comment]
How have you installed ffmpeg and what's the version?

When I test PyAV locally, `test_subtitles.py` passes on my machine, although `test_file_probing.py` and `test_python_io.py` fails.

platform darwin -- Python 3.12.0, pytest-7.4.3, pluggy-1.3.0
ffmpeg HEAD-84e400a_1 via Homebrew

```
FAILED tests/test_file_probing.py::TestVideoProbe::test_stream_probing - AssertionError: Fraction(25, 1) is not None
FAILED tests/test_file_probing.py::TestVideoProbeCorrupt::test_stream_probing - AssertionError: Fraction(25, 1) != None
FAILED tests/test_python_io.py::TestPythonIO::test_writing_to_custom_io_dash - av.error.InvalidDataError: [Errno 1094995529] Invalid data found when processing input: 'custom_io_output.mpd'
```

This is not unexpected because i'm using HEAD ffmpeg (6.1) instead of 6.0 release.

**Comment by yu17** [comment]
Apologies for the confusion. I built from AUR and thought we were still on ffmpeg-4.4. Didn't even check the build script about that.

Apparently ffmpeg-6.0 behaves differently. The commits I referred to either were unrelated or won't affect the tests.

I just tested again on the ffmpeg-6.1 (release, not HEAD). Interestingly, I had the same two errors in `test_file_probing.py`, but `test_python_io.py` passed.

Closing this issue. Again sorry for the trouble!
